src/js/mqtt.js

```diff
@@ -52,6 +54,7 @@ function MqttClient(endpoint, options) {
   this._lastConnectTime = 0;
   this._msgId = 0;
   this._ttl = null;
+  this._timeoutTTL = null;
```

Comment:
naming suggestion: `_keepAliveTimer` and `_keepAliveTimeout`

Reply:
+1 on `_keepAliveTimer`. The `TTL` might be a number usually, "timer" would be better for naming :)

Reply:
haha, ttl was named by you @yorkie 
let me fix it